march_shared_classes/src/march_shared_classes/gait/subgait.py

```diff
@@ -163,6 +163,7 @@ def to_subgait_msg(self):
         subgait_msg.trajectory = self._to_joint_trajectory_msg()
         subgait_msg.setpoints = self._to_setpoints_msg()
         subgait_msg.description = str(self.description)
+        subgait_msg.version = str(self.version)
```

Comment:
```suggestion
        subgait_msg.version = self.version
```
string should not be necessary